hypothesis-python/src/hypothesis/reporting.py

```diff
@@ -54,7 +54,7 @@ def current_verbosity():
 def to_text(textish):
     if inspect.isfunction(textish):
         textish = textish()
-    if isinstance(textish, bytes):
+    if isinstance(textish, bytes):  # noqa: B1
```

Comment:
This one could be `compat.binary_type` instead of `bytes` - Python 2 has `str.decode`.